src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -113,6 +150,662 @@ public FloatingPointInfo(ushort denormalMantissaBits, ushort exponentBits, int m
+        0xb2c71d5bca9023f8, 0x743e20e9ef511012,
+        0xdf78e4b2bd342cf6, 0x914da9246b255416,
+        0x8bab8eefb6409c1a, 0x1ad089b6c2f7548e,
+        0xae9672aba3d0c320, 0xa184ac2473b529b1,
+        0xda3c0f568cc4f3e8, 0xc9e5d72d90a2741e,
+        0x8865899617fb1871, 0x7e2fa67c7a658892,
+        0xaa7eebfb9df9de8d, 0xddbb901b98feeab7,
+        0xd51ea6fa85785631, 0x552a74227f3ea565,
+        0x8533285c936b35de, 0xd53a88958f87275f,
+        0xa67ff273b8460356, 0x8a892abaf368f137,
+        0xd01fef10a657842c, 0x2d2b7569b0432d85,
+        0x8213f56a67f6b29b, 0x9c3b29620e29fc73,
+        0xa298f2c501f45f42, 0x8349f3ba91b47b8f,
+        0xcb3f2f7642717713, 0x241c70a936219a73,
+        0xfe0efb53d30dd4d7, 0xed238cd383aa0110,
+        0x9ec95d1463e8a506, 0xf4363804324a40aa,
+        0xc67bb4597ce2ce48, 0xb143c6053edcd0d5,
+        0xf81aa16fdc1b81da, 0xdd94b7868e94050a,
+        0x9b10a4e5e9913128, 0xca7cf2b4191c8326,
+        0xc1d4ce1f63f57d72, 0xfd1c2f611f63a3f0,
+        0xf24a01a73cf2dccf, 0xbc633b39673c8cec,
+        0x976e41088617ca01, 0xd5be0503e085d813,
+        0xbd49d14aa79dbc82, 0x4b2d8644d8a74e18,
+        0xec9c459d51852ba2, 0xddf8e7d60ed1219e,
+        0x93e1ab8252f33b45, 0xcabb90e5c942b503,
+        0xb8da1662e7b00a17, 0x3d6a751f3b936243,
+        0xe7109bfba19c0c9d, 0xcc512670a783ad4,
+        0x906a617d450187e2, 0x27fb2b80668b24c5,
+        0xb484f9dc9641e9da, 0xb1f9f660802dedf6,
+        0xe1a63853bbd26451, 0x5e7873f8a0396973,
+        0x8d07e33455637eb2, 0xdb0b487b6423e1e8,
+        0xb049dc016abc5e5f, 0x91ce1a9a3d2cda62,
+        0xdc5c5301c56b75f7, 0x7641a140cc7810fb,
+        0x89b9b3e11b6329ba, 0xa9e904c87fcb0a9d,
+        0xac2820d9623bf429, 0x546345fa9fbdcd44,
+        0xd732290fbacaf133, 0xa97c177947ad4095,
+        0x867f59a9d4bed6c0, 0x49ed8eabcccc485d,
+        0xa81f301449ee8c70, 0x5c68f256bfff5a74,
+        0xd226fc195c6a2f8c, 0x73832eec6fff3111,
+        0x83585d8fd9c25db7, 0xc831fd53c5ff7eab,
+        0xa42e74f3d032f525, 0xba3e7ca8b77f5e55,
+        0xcd3a1230c43fb26f, 0x28ce1bd2e55f35eb,
+        0x80444b5e7aa7cf85, 0x7980d163cf5b81b3,
+        0xa0555e361951c366, 0xd7e105bcc332621f,
+        0xc86ab5c39fa63440, 0x8dd9472bf3fefaa7,
+        0xfa856334878fc150, 0xb14f98f6f0feb951,
+        0x9c935e00d4b9d8d2, 0x6ed1bf9a569f33d3,
+        0xc3b8358109e84f07, 0xa862f80ec4700c8,
+        0xf4a642e14c6262c8, 0xcd27bb612758c0fa,
+        0x98e7e9cccfbd7dbd, 0x8038d51cb897789c,
+        0xbf21e44003acdd2c, 0xe0470a63e6bd56c3,
+        0xeeea5d5004981478, 0x1858ccfce06cac74,
+        0x95527a5202df0ccb, 0xf37801e0c43ebc8,
+        0xbaa718e68396cffd, 0xd30560258f54e6ba,
+        0xe950df20247c83fd, 0x47c6b82ef32a2069,
+        0x91d28b7416cdd27e, 0x4cdc331d57fa5441,
+        0xb6472e511c81471d, 0xe0133fe4adf8e952,
+        0xe3d8f9e563a198e5, 0x58180fddd97723a6,
+        0x8e679c2f5e44ff8f, 0x570f09eaa7ea7648};
+
+        internal const int SmallestPowerOfFive = -342;
+        internal const int LargestPowerOfFive = 308;
```

Comment:
I'd presume these constants are tied to the constants from the `FloatingPointInfo Double` entry? Should they live next to eachother or be directly tied together to ensure they stay in sync?

Reply:
got rid of thoses.  I plugged SmallestPowerofFive in the function itself with a note instead.